plugin.py

```diff
@@ -106,12 +123,12 @@ def find_package_dependency_dirs(self, py_ver: Tuple[int, int] = (3, 3)) -> List
 
         # move the "Packages/" to the last
         # @see https://github.com/sublimelsp/LSP-pyright/pull/26#discussion_r520747708
-        packages_path = sublime.packages_path()
-        dep_dirs.remove(packages_path)
-        dep_dirs.append(packages_path)
+        dep_dirs.remove(SUBLIME_PACKAGES_PATH)
+        dep_dirs.append(SUBLIME_PACKAGES_PATH)
 
-        # sublime stubs - add as first
-        dep_dirs.insert(0, os.path.join(self.package_storage(), "resources", "typings", "sublime_text"))
+        if py_ver == (3, 3):
+            # sublime stubs - add as first
+            dep_dirs.insert(0, os.path.join(self.package_storage(), "resources", "typings", "sublime_text"))
```

Comment:
I wonder how good is the builtin type annotations work for py38. I see some type annotation are missing in method signatures. Especially for `__init__(...)`. But yes, stubs are a little outdated as it's for like ST 4126.